recipe/meta.yaml

```diff
@@ -9,9 +9,10 @@ source:
   sha256: 77a636bdc08c7668a15951894548c527f0c8c5c2abc86cb850de17551af51e3e
 
 build:
-  number: 2
+  number: 3
   script:
     - export LDFLAGS="-headerpad_max_install_names"  # [osx]
```

Comment:
We should be able to drop this line if we add `toolchain` to `build`. I should have done that a long time ago.

Reply:
I don't know what this comment means but it seems to be very suitable for a separate PR... =P

Reply:
Yeah I can address it in a follow-up PR.